code/_helpers/text.dm

```diff
@@ -227,7 +236,7 @@
 
 //Returns a string with the first element of the string capitalized.
 /proc/capitalize(var/t as text)
-	return uppertext(copytext(t, 1, 2)) + copytext(t, 2)
+	return uppertext(copytext_char(t, 1, 2)) + copytext_char(t, 2)
```

Comment:
This looks like the only thing that's actually doing anything, what's that other thing for?